### ISSUE TITLE
iOS build fails on 3.19.0

### ISSUE THREAD

**Issue by Coolcat13** [Anyline/anyline-ocr-cordova-module#16]
Hi there,

With no other change than this plugin's version to 3.19.0, our ios builds are now failing with the following message - can you help? If I revert to 3.18.3, no such problem occurs. Is 3.19.0 supposed to fix the issue on ios 9? (See #13) - Building with xcode 8.2.1 (have to)

Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_AnylineDocumentModuleView", referenced from:
      objc-class-ref in AnylineDocumentScanViewController.o
  "_OBJC_CLASS_$_AnylineEnergyModuleView", referenced from:
      objc-class-ref in AnylineEnergyScanViewController.o
  "_kCodeTypeDataMatrix", referenced from:
      ___64-[AnylineEnergyScanViewController barcodeFormatForNativeString:]_block_invoke in AnylineEnergyScanViewController.o
  "_kCodeTypePDF417", referenced from:
      ___64-[AnylineEnergyScanViewController barcodeFormatForNativeString:]_block_invoke in AnylineEnergyScanViewController.o
  "_kCodeTypeCode39", referenced from:
      ___64-[AnylineEnergyScanViewController barcodeFormatForNativeString:]_block_invoke in AnylineEnergyScanViewController.o
  "_OBJC_CLASS_$_AnylineOCRModuleView", referenced from:
      objc-class-ref in AnylineOCRScanViewController.o
  "_kCodeTypeCode128", referenced from:
      ___64-[AnylineEnergyScanViewController barcodeFormatForNativeString:]_block_invoke in AnylineEnergyScanViewController.o
  "_kCodeTypeCode93", referenced from:
      ___64-[AnylineEnergyScanViewController barcodeFormatForNativeString:]_block_invoke in AnylineEnergyScanViewController.o
  "_kCodeTypeEAN13", referenced from:
      ___64-[AnylineEnergyScanViewController barcodeFormatForNativeString:]_block_invoke in AnylineEnergyScanViewController.o
  "_kCodeTypeITF", referenced from:
      ___64-[AnylineEnergyScanViewController barcodeFormatForNativeString:]_block_invoke in AnylineEnergyScanViewController.o
  "_kCodeTypeUPCE", referenced from:
      ___64-[AnylineEnergyScanViewController barcodeFormatForNativeString:]_block_invoke in AnylineEnergyScanViewController.o
  "_kCodeTypeEAN8", referenced from:
      ___64-[AnylineEnergyScanViewController barcodeFormatForNativeString:]_block_invoke in AnylineEnergyScanViewController.o
  "_OBJC_CLASS_$_AnylineLicensePlateModuleView", referenced from:
      objc-class-ref in AnylineLicensePlateViewController.o
  "_OBJC_CLASS_$_ALUIConfiguration", referenced from:
      objc-class-ref in AnylineSDKPlugin.o
  "_kCodeTypeAztec", referenced from:
      ___64-[AnylineEnergyScanViewController barcodeFormatForNativeString:]_block_invoke in AnylineEnergyScanViewController.o
  "_OBJC_CLASS_$_AnylineMRZModuleView",
 referenced from:
      objc-class-ref in AnylineMRZScanViewController.o
  "_kCodeTypeQR", referenced from:
      ___64-[AnylineEnergyScanViewController barcodeFormatForNativeString:]_block_invoke in AnylineEnergyScanViewController.o
  "_OBJC_CLASS_$_ALOCRConfig", referenced from:
      objc-class-ref in AnylineOCRScanViewController.o
  "_OBJC_CLASS_$_AnylineBarcodeModuleView", referenced from:
      objc-class-ref in AnylineBarcodeScanViewController.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

** ARCHIVE FAILED **

The following build commands failed:
	Ld /Users/XXXXX/Library/Developer/Xcode/DerivedData/XXXXX-dgkqmnfnlrylthfelvaxusdeeksm/Build/Intermediates/ArchiveIntermediates/XXXXX/IntermediateBuildFilesPath/X.build/DebugXXXXX-iphoneos/XXXXX.build/Objects-normal/arm64/XXXXXX normal arm64
(1 failure)

Error: Error code 65 for command: xcodebuild with args: -xcconfig,/Users/XXXXX/documents/apps/phonegap/platforms/ios/cordova/build-debug.xcconfig,-workspace,XXXXX.xcworkspace,-scheme,XXXXX,-configuration,Debug,-destination,generic/platform=iOS,-archivePath,XXXXX.xcarchive,archive,CONFIGURATION_BUILD_DIR=/Users/XXXXX/documents/apps/phonegap/platforms/ios/build/device,SHARED_PRECOMPS_DIR=/Users/XXXXX/documents/apps/phonegap/platforms/ios/build/sharedpch

**Comment by JonasLaux** [comment]
Hi, 
yes, we currently have problems with npm. Our SDK isn't uploaded right. 
For a quickfix, just get our SDK [here](https://github.com/Anyline/anyline-ocr-examples-ios/releases/download/v3.19.0/AnylineSDK_iOS_3.19.0.zip) and put the content of the framework folder into node_modules/anyline-ocr-cordova-module/src/ios/anyline_sdk of your project

**Comment by Coolcat13** [comment]
Unfortunately that didn't help - we didn't actually have a anyline-ocr-cordova-module folder in node_modules so created it and copied the framework folder into the path you gave, but the same error occurs. Anything else we can try? Should the framework not be copied into the platform/ios folder?

**Comment by Coolcat13** [comment]
Hi, the build now finishes but none of the plugins get included in the build which means nothing works...any idea? Does copying the files manually potentially break something in cordova? (again if we rebuild with version 3.18.3 no problem)

Note: the new anyline sdk files had to be copied to the plugins folder not the node_modules folder (which doesn't have an anyline folder)

**Comment by JonasLaux** [comment]
Hi,
we switched to Cocoapods. Please update to 3.19.1.
After you downloaded the plugin, go to platforms/ios and run "pod update".

**Comment by Coolcat13** [comment]
Yes that worked thanks...however after all that, it still crashes on iOS 9...any update on that issue? (See #13)

**Comment by JonasLaux** [comment]
with the same crashlog?

**Comment by Coolcat13** [comment]
Actually no, a different one, butI it looks like it works fine when generating a full build (cordova build instead of cordova run) - let me try again and confirm.

**Comment by Coolcat13** [comment]
I can confirm the build now works using v 3.19.1, just not when we do a cordova run, but we can live with that. Thanks for your help.
